solr/bin/solr

```diff
@@ -1076,10 +1076,8 @@ if [ "${SOLR_HOME:0:${#EXAMPLE_DIR}}" = "$EXAMPLE_DIR" ]; then
   SOLR_LOGS_DIR="$SOLR_HOME/../logs"
 fi
 
-# Set the logging manager by default, so that Lucene JUL logs are included with Solr logs.
-LOG4J_CONFIG=("-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager")
 if [ -n "${LOG4J_PROPS:-}" ]; then
-  LOG4J_CONFIG+=("-Dlog4j.configurationFile=$LOG4J_PROPS")
+  LOG4J_CONFIG=("-Dlog4j.configurationFile=$LOG4J_PROPS")
 fi
```

Comment:
Wondering if `else` is needed to ensure `LOG4J_CONFIG` is always initialised?
```suggestion
else
  LOG4J_CONFIG=
fi
```

Reply:
Do we have a bats test that would catch the `LOG4J_CONFIG` not being initialized and show if this is needed?  

There is a lot of tricky logic in the start command, and I'd love to have more tests so that as we refactor/change it we have more confidence that the tests are meaningful!

Reply:
I added a commit to initialize LOG4J_CONFIG=() which is what it was originally https://github.com/apache/solr/pull/1765/files#diff-49c29c8f653341f48008c38f5f2cf970fa430cdca29181c819d7bdcfcc722980